pkg/nsx/util/utils.go

```diff
@@ -29,6 +28,14 @@ type ErrorDetail struct {
 	Details            string
 }
 
+// PortAddress is used when named port is specified.
+type PortAddress struct {
+	// Port is the port number.
+	Port int `json:"port"`
+	// IPs is a list of IPs.
```

Comment:
IPs is a list of IPs associated to port number.

Reply:
ok